### DISCUSSION TITLE
벌칙 업데이트 해주세요.

### DISCUSSION THREAD

**Discussion by on-Sync** [code-etc/Documents#2] [General]
@daehwan2 @yunjin-kim 님 벌칙 업데이트 부탁드려요.

- 벌칙대상
  - 지각 벌칙
  - 불참 벌칙

> 최소 3시간 전에 미공지하지 않은 인원에 대한 벌칙입니다.
> 아래 링크에 기록해주세요.

[의무 불이행에 대한 처우](https://github.com/code-etc/Documents/tree/main/%EC%9D%B4%EC%83%81%ED%98%95%EC%9B%94%EB%93%9C%EC%BB%B5/%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8%EA%B4%80%EB%A6%AC/%EC%A4%80%EC%88%98%EC%82%AC%ED%95%AD#%EC%9D%98%EB%AC%B4-%EB%B6%88%EC%9D%B4%ED%96%89%EC%97%90-%EB%8C%80%ED%95%9C-%EC%B2%98%EC%9A%B0)

**Comment by daehwan2** [comment]
업데이트 완료했습니다
